osquery/tables/applications/browser_utils.cpp

```diff
@@ -149,20 +185,32 @@ QueryData genChromeBasedExtensions(QueryContext& context,
       }
 
       // For each profile list each extension in the Extensions directory.
-      std::vector<std::string> extensions;
       for (const auto& profile : profiles) {
+        std::vector<std::string> extensions = {};
         listDirectoriesInDirectory(profile, extensions);
-      }
 
-      // Generate an addons list from their extensions JSON.
-      std::vector<std::string> versions;
-      for (const auto& extension : extensions) {
-        listDirectoriesInDirectory(extension, versions);
-      }
+        if (extensions.empty()) {
+          continue;
+        }
+
+        auto profile_path = fs::path(profile).parent_path().parent_path();
+
+        std::string profile_name;
+        auto status = getChromeProfileName(profile_name, profile_path);
+        if (!status.ok()) {
+          LOG(WARNING) << status.getMessage();
```

Comment:
May I ask you to add a prefix for this log line? Smth like: "Getting chrome profile name failed: " ?